docs/source/analysis/scipp.rst

```diff
@@ -0,0 +1,346 @@
+   * x                         float64              [m]  (x)  [-9.6e-06, -8.8e-06, ..., 9.6e-06, 1.04e-05]
+   * y                         float64              [m]  (y)  [-1e-05, -9.2e-06, ..., 9.2e-06, 1e-05]
+   * z                         float64              [m]  (z)  [4.7e-06, 4.8e-06, ..., 2.46e-05, 2.47e-05]
+   Data:
+                               float64            [V/m]  (x, y, z)  [-1.08652e+08, -1.9758e+08, ..., 0, 0]
+
+.. code:: python
+
+   # For scalar fields just omit the second argument:
+   print(data_loader.get_field("rho", iteration=200))
+
+::
+
+   <scipp.DataArray>
+   Dimensions: Sizes[x:26, y:26, z:201, ]
+   Coordinates:
+   * t                         float64              [s]  ()  6.56942e-14
+   * x                         float64              [m]  (x)  [-1e-05, -9.2e-06, ..., 9.2e-06, 1e-05]
+   * y                         float64              [m]  (y)  [-1e-05, -9.2e-06, ..., 9.2e-06, 1e-05]
+   * z                         float64              [m]  (z)  [4.7e-06, 4.8e-06, ..., 2.46e-05, 2.47e-05]
+   Data:
+                               float64           [mC/L]  (x, y, z)  [-7169.01, -7526.4, ..., 3.16049e-11, 1.22782e-11]
+
+Plotting
+--------
+
+We can't directly plot 3D data. But we can for example select a slice.
+For that we can use a helper function ``pmdsc.closest`` to get the
+closets index, since ``scipp`` requires exact match. You can read more
+about indexing ``scipp`` arrays in ``scipp``'s documentation.
+
+.. code:: python
+
+   slicing_idx = pmdsc.closest(Ex, "x", 2 * sc.Unit("um"))
+   Ex_slice = Ex["x", slicing_idx]
+   print(Ex_slice)
+
+::
+
+   <scipp.DataArray>
+   Dimensions: Sizes[y:26, z:201, ]
+   Coordinates:
+   * t                         float64              [s]  ()  6.56942e-14
+     x                         float64              [m]  ()  2.4e-06
+   * y                         float64              [m]  (y)  [-1e-05, -9.2e-06, ..., 9.2e-06, 1e-05]
+   * z                         float64              [m]  (z)  [4.7e-06, 4.8e-06, ..., 2.46e-05, 2.47e-05]
+   Data:
+                               float64            [V/m]  (y, z)  [4.86614e+08, 6.67018e+08, ..., 0, 0]
+
+.. code:: python
+
+   Ex_slice.plot()
+
+.. figure:: README_15_0.svg
+
+.. code:: python
+
+   # We can also plot line plots:
+   Ex_line = Ex_slice["z", pmdsc.closest(Ex_slice, "z", 1.4e-5 * sc.Unit("m"))]
+   print(Ex_line)
+
+::
+
+   <scipp.DataArray>
+   Dimensions: Sizes[y:26, ]
+   Coordinates:
+   * t                         float64              [s]  ()  6.56942e-14
+     x                         float64              [m]  ()  2.4e-06
+   * y                         float64              [m]  (y)  [-1e-05, -9.2e-06, ..., 9.2e-06, 1e-05]
+     z                         float64              [m]  ()  1.4e-05
+   Data:
+                               float64            [V/m]  (y)  [3.46069e+07, -2.94134e+07, ..., 8.89353e+06, -4.32182e+07]
+
+.. code:: python
+
+   Ex_line.plot()
+
+.. figure:: README_17_0.svg
```

Comment:
Can you potentially upload the four `.svg` images to a public GH gist and use them by URL? That way, we do not add larger binary artifacts to our git repo here, which keeps it small on `git clone` for CMake superbuilds.

Reply:
SVG files are not binary, these each are text files with each ~800 lines, less than many of our source files.
I tried putting them on a [Gist](https://gist.github.com/franzpoeschel/de7e3ef4b7b837c45387ffafac10f961), but it seems that Github does not hand out usable raw links.